react-motions/src/shake/shake.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './styles'
+
+function Shake({ children, duration, fillMode, infinite }) {
```

Comment:
I was wondering if `duration` isn't the best inference to `infinite`, like:
`<Animation duration={Infinity}/>`

What do you think?

Reply:
the `animationIterationCount` attribute only works if u have an` animationDuration`. So I think it's better to be separated so that you can change the time even when u dont.

the infinite is: repeats the animation each time it ends (duration).

Reply:
True, you're right.